skll/config.py

```diff
@@ -474,6 +474,26 @@ def _parse_config_file(config_path):
     # using the defaults?
     do_grid_search = config.getboolean("Tuning", "grid_search")
 
+    # Check `fixed_parameter_list`/`param_grid_list` if `do_grid_search`
+    # is True
+    if do_grid_search:
+        for learner, fixed_params, params in zip(fixed_parameter_list,
+                                                 param_grid_list,
+                                                 learners):
+            overlap_params = set(fixed_params).intersection(set(params))
+            if overlap_params:
+                logger.warning('Both "fixed_parameters" and "param_grids" were'
+                               'specified or the default values were used for '
+                               '{}, which resulted in conflicts between '
```

Comment:
You mean the situation where somebody specifies some fixed parameters but leaves on grid search (without specifying the grid)? I think the warning message could be more general, something like: "There is a conflict between the fixed parameters specified and the parameters to be searched during grid search. The fixed parameters will be used."?